paywall/src/data-iframe/Mailbox.ts

```diff
@@ -213,6 +212,9 @@ export default class Mailbox {
     })
   }
 
+  /**
+   * TODO: consider moving all this to the constructor... this will avoid race conditions
```

Comment:
:+1: It looks like nothing here needs to be waited on, so it'll be a good future move to do this

Reply:
(although may not be needed in the long run since we're moving all of this stuff)

Reply:
yeah :/